web/lattice_main/render_to_file.py

```diff
@@ -0,0 +1,31 @@
+#### Stand-alone script version of render. (not yet sure why the CSS won't render)
```

Comment:
I wrote this as a stand-alone script, however, I'm not sure what output you are expecting, and the CSS doesn't render. I also integrated it into a view that is callable by URL. Maybe we want to integrate saving html outputs into the GUI? Looking for guidance on how you were using this in your debugging.

Reply:
Thank you keelan this is very helpful I think the reason why the css won't render is because it's served on another domain

Reply:
Maybe this helps https://stackoverflow.com/questions/22162027/how-do-i-generate-a-static-html-file-from-a-django-template 

Anyways the reason why we care is for debugging, so we can run the `debug_*` files directly from the ide with the debugger, without having to start the server with the debugger enabled

Reply:
So no need for integration of this tool in the gui, it's just for debugging

Reply:
Ok, I'll remove it from the Views then